api/models/article-models.js

```diff
@@ -18,3 +19,17 @@ exports.fetchArticleId = (params) => {
     }
   });
 };
+
+exports.fetchAllArticles = (order) => {
+  const orderAllowed = ["ASC", "DESC"];
+  if (!orderAllowed.includes(order)) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  const queryString = format(
+    `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments USING (article_id) GROUP BY articles.article_id ORDER BY articles.created_at %s`,
+    [order]
+  );
```

Comment:
You don't need to use format here - format is for doing large queries for loads of data. Instead of using format just use $1 for created_at